docs/development/reconciliation_en.md

```diff
@@ -0,0 +1,61 @@
+# How reconciliation works in the `k8s-dogu-operator`.
+
+The Dogu operator receives a Dogu resource with the target state from the Kubernetes API and thus passes it the
+Control to bring about the target state (called reconciliation). This can be, for example, Dogu-.
+Reinstallation, Upgrade or also Dogu Uninstallation (called "operation").
+
+## Requeueing to continue operations
+
+During an operation, external circumstances (such as errors or long waiting times) may occur. So that the Dogu operator
+does not stand still for this time, there is the possibility to stop the Reconcile function for the time being, to be executed again later.
+to be executed again later.
```

Comment:
to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. to be executed again later. :rofl: